app/controllers/carts_controller.rb

```diff
@@ -1,4 +1,14 @@
 class CartsController < ApplicationController
+  def create
+    update_cart
+    flash[:notice] = 'Added to Cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.